elastic/kibana
# [Maps] Update EMS Landing Page link to root or vX.Y

Fixes #161791

Updates the link to the EMS Landing Page website in the Kibana Maps UI to point to the root directory at https://maps.elastic.co when the build is `serverless` and to the correct `https://maps.elastic.co/vX.Y` for `traditional` builds.

To test:

* When running with `yarn start`
  * Check the link in the source details of an EMS Administrative boundaries dataset in the Maps Application
* When running with `yarn start --serverless=es`
  * Check the source details of an EMS Administrative boundaries dataset in the Maps Application

:warning: **Note**: This change does not affect the link in the Tutorial for the Elastic Maps Service since it only touches the EMS Client creation and not the `EMSSettings` that happens synchronously. This is OK since that tutorial should point to the latest Landing Page revision to offer all the data for our users.
